src/test/java/com/sparkplatform/api/core/ClientTest.java

```diff
@@ -45,14 +45,6 @@ public void setup(){
 		c = new SimpleClient(config, conn, conn);
 		sample = new HashMap<String, String>();
 		sample.put("Optimus",   "semi");
-		sample.put("Soundwave", "walkman");
-		sample.put("Starscream", "F-15 Eagle");
-		sample.put("Megatron",  "Walther P-38");
-	}
```

Comment:
I added multiple parameters to two tests below that test the signature and param string as built for the request.  However, the other tests get hung up as the order of these parameters in the string is not guaranteed.